packages/ui/src/theme/css/component/stepperField.scss

```diff
@@ -42,31 +42,29 @@
   );
 }
 
-.amplify-stepperfield__button--decrease {
+.amplify-stepperfield__button--decrease,
+.amplify-stepperfield__button--increase {
   background-color: var(
     --amplify-components-stepperfield-button-background-color
   );
-  &[data-invalid='true'] {
-    border-inline-end: none;
+  &--disabled {
+    background-color: var(--amplify-colors-background-disabled);
```

Comment:
I think we should use `var(--amplify-components-stepperfield-button-disabled-background-color);` but update its token value instead of using the inherited value from `Button` - https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/components/stepperField.ts#L57

Reply:
Done!